Error while checking build/O2/fullCI for 0d625ba7ca0f476dd243c3819d8286af88b109bc at 2023-02-10 12:16:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/10739-slc8_x86-64/0/GPU/TPCFastTransformation/TPCFastTransform.h(120): error: calling a __device__ function("o2::utils::DebugStreamer::flush() const") from a __host__ function("flushStreamer") is not allowed
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10739/0d625ba7ca0f476dd243c3819d8286af88b109bc/build_O2_fullCI/pretty.html).

--- a/GPU/TPCFastTransformation/TPCFastTransform.h
+++ b/GPU/TPCFastTransformation/TPCFastTransform.h
@@ -117,7 +117,7 @@ class TPCFastTransform : public FlatObject
 #endif
 
   /// write output of streamer to file
-  void flushStreamer() { mStreamer.flush(); }
+  GPUd() void flushStreamer() { mStreamer.flush(); }
 
   /// _____________  FlatObject functionality, see FlatObject class for description  ____________
 

